Libraries/libutil/Sources/MemoryProcessContext.cpp

```diff
@@ -32,15 +34,16 @@ MemoryProcessContext(
 }
 
 MemoryProcessContext::
-MemoryProcessContext(ProcessContext const &processContext) :
+MemoryProcessContext(ProcessContext const *processContext) :
```

Comment:
These are usually passed around by pointer since they have virtual methods, and it's non-obvious that calling via references goes through the vtable. Same reasoning here, just forgot that at first.